.github/workflows/workflow.yml

```diff
@@ -44,96 +36,91 @@ jobs:
   # - every PR
   #
 
-  validate_go_deps:
-    name: Validate go deps
+  go_dependencies:
+    name: Go dependencies
     runs-on: ubuntu-18.04
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
-    # for debugging
-    - name: Dump env
-      run: |
-        env | sort
-    - name: Dump GitHub context
-      env:
-        GITHUB_CONTEXT: ${{ toJson(github) }}
-      run: echo "$GITHUB_CONTEXT"
-    - name: Dump job context
-      env:
-        JOB_CONTEXT: ${{ toJson(job) }}
-      run: echo "$JOB_CONTEXT"
-    - name: Validate go deps
-      run: |
-        . bin/_tag.sh
-        for f in $( grep -lR --include=Dockerfile\* go-deps: . ) ; do
-          validate_go_deps_tag $f
-        done
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Dump env
+        run: env | sort
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      - name: Dump job context
+        env:
+          JOB_CONTEXT: ${{ toJson(job) }}
+        run: echo "$JOB_CONTEXT"
+      - name: Validate go deps
+        run: |
+          . bin/_tag.sh
+          for f in $( grep -lR --include=Dockerfile\* go-deps: . ) ; do
+            validate_go_deps_tag $f
+          done
 
   go_unit_tests:
     name: Go unit tests
     runs-on: ubuntu-18.04
     container:
       image: golang:1.13.4
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
-    - name: Go unit tests
-      env:
-        GITCOOKIE_SH: ${{ secrets.GITCOOKIE_SH }}
-      run: |
-        echo "$GITCOOKIE_SH" | bash
-        # TODO: validate bin/protoc-go.sh does not dirty the repo
-        go test -cover -race -v -mod=readonly ./...
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Go unit tests
+        env:
+          GITCOOKIE_SH: ${{ secrets.GITCOOKIE_SH }}
+        run: |
+          echo "$GITCOOKIE_SH" | bash
+          # TODO: validate bin/protoc-go.sh does not dirty the repo
+          go test -cover -race -v -mod=readonly ./...
 
   go_lint:
     name: Go lint
     runs-on: ubuntu-18.04
     container:
       image: golang:1.13.4
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
-    - name: Go lint
-      env:
-        GITCOOKIE_SH: ${{ secrets.GITCOOKIE_SH }}
-        # prevent OOM
-        GOGC: 20
```

Comment:
No longer needed?

Reply:
I removed this on most runs of workflow tests on my fork for both the forked PRs testing as well as testing for this.

I have not run into an issue with it since, and I think it's just carried over from a time when it was required. I know it's strictly necessary to *remove*, but I have not been able to see a case where it's needed.